spec/mailers/order_delivery_mailer_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe OrderDeliveryMailer, type: :mailer do
+  describe 'Order delivery confirmation' do
+    let(:order) { create(:order) }
+    let(:mail) { described_class.with(order: order).delivery_email.deliver_now }
+
+    it 'checking the headers' do
```

Comment:
suggestion for naming specs
```suggestion
    it 'has proper headers' do
```

Reply:
fixed